docs/usage/faq-troubleshooting.md

```diff
@@ -0,0 +1,9 @@
+# Troubleshooting and FAQs
+
+## I get a warning during centrifuge_kreport process with exit status 255
```

Comment:
```suggestion
## I get a warning during centrifuge_report process with exit status 255
```
not sure if this is a typo or not

Reply:
Not a typo, it's called `centrifuge_kreport`